README.md

```diff
@@ -14,20 +14,27 @@ documented in the `.msg` or `.srv` file associated with that interface.
 ROS messages and services used across many ROS packages in the project.
 
 ### Project-wide External Interfaces
-
+<!---
+Formatting reminder:
+1. Keep 'Type' column organized alphabetically
+2. 'mock' version of topic should be after its common counterpart
+--->
 | Topic                  | Type           | Publisher                | Subscriber(s)                               |
 | ---------------------- | -------------- | ------------------------ | ------------------------------------------- |
-| `ais_ships`            | AISShips       | AIS Receiver             | Local Pathfinding                           |
-| `batteries`            | Batteries      | CAN Transceiver          | Local Transceiver                           |
+| `ais_ships`            | AISShips       | CanTrxRosIntf            | Local Pathfinding, Local Transceiver        |
+| `mock_ais_ships`       | AISShips       | Mock AIS                 | CanSimIntf                                  |
+| `batteries`            | Batteries      | CanTrxRosIntf            | Local Transceiver                           |
 | `desired_heading`      | DesiredHeading | Local Pathfinding        | Controller, Boat Simulator                  |
-| `data_sensors`         | GenericSensors | CAN Transceiver          | Local Transceiver                           |
-| `global_path`          | Path           | Local Transceiver        | Local Pathfinding                           |
-| `gps`                  | GPS            | CAN Transceiver          | Local Transceiver, Local Pathfinding        |
+| `data_sensors`         | GenericSensors | CanTrxRosIntf            | Local Transceiver                           |
+| `gps`                  | GPS            | CanTrxRosIntf            | Local Transceiver, Local Pathfinding        |
+| `mock_gps`             | GPS            | Boat Simulator           | CanSimIntf                                  |
 | `local_path_data`      | LPathData      | Local Pathfinding        | Local Transceiver                           |
-| `mock_gps`             | GPS            | Boat Simulator           | CAN Transceiver                             |
-| `filtered_wind_sensor` | WindSensor     | CAN Transceiver          | Local Transceiver, Local Pathfinding        |
-| `mock_wind_sensors`    | WindSensors    | Boat Simulator           | CAN Transceiver                             |
-| `wind_sensors`         | WindSensors    | CAN Transceiver          | Local Transceiver                           |
+| `global_path`          | Path           | Local Transceiver        | Local Pathfinding                           |
+| `sail_cmd`             | SailCmd        | Controller               | CanTrxRosIntf                               |
+| `sim_sail_cmd`         | SailCmd        | CanSimIntf               | Simulator                                   |
```

Comment:
@colinli02 What is the difference between `CanSimIntf`, `Simulator`, and `Boat Simulator`? If `Simulator` and `Boat Simulator` are meant to be equivalent, can you change all instances of `Simulator` to `Boat Simulator` to make it consistent.

I also noticed that you have another topic called `sim_sail_cmd`. I suppose that the boat simulator should subscribe to that topic to simulate getting that data over CAN? If that is the case, the boat simulator currently subscribes to the topic `desired_heading`. I think we should also make another topic called `sim_desired_heading` to simulate the low level controls getting the desired heading over CAN, and then change the table in the README accordingly. @patrick-5546 what do you think about this?

Reply:
Also, let's keep the boat simulator subscribed to `sail_cmd` and `desired_heading` because I want to add a configuration option in the simulator to bypass the CAN simulation. This is so we can test the simulator until the simulation interface is ready.

Reply:
> If Simulator and Boat Simulator are meant to be equivalent, can you change all instances of Simulator to Boat Simulator to make it consistent.

Agreed

> I think we should also make another topic called sim_desired_heading

Will this also be published by CanSimIntf and subscribed by Boat Simulator? I think we should just have one msg between every publisher and subscriber

> Also, let's keep the boat simulator subscribed to sail_cmd and desired_heading

Sounds good

Reply:
Can you or one of your members make these changes in this PR @DFriend01? They are simulator related

Reply:
> Will this also be published by CanSimIntf and subscribed by Boat Simulator? I think we should just have one msg between every publisher and subscriber

Yes, the idea is to have the CanSimIntf subscribe to the topics in ROS, and then it publishes to the simulation interfaces `sim_desired_heading` and `sim_sail_cmd`, and the boat simulator would subscribe to these topics.

If the boat simulator bypasses the can simulator interface, then it just subscribes directly to `sail_cmd` and `desired_heading`. Why do we want to bypass? It's more of a temporary thing until the interface is working, but in the meantime, we directly subscribe to the topics published to by the ROS nodes.

Reply:
@patrick-5546 When you say make the changes, do you mean change the publish/subscriber flow?

Reply:
> Yes, the idea is to have the CanSimIntf subscribe to the topics in ROS sail_cmd and desired_heading, and then it publishes to the simulation interfaces sim_desired_heading and sim_sail_cmd, and the boat simulator would subscribe to these topics.

How about just having one msg, `sim_can_signals`

> When you say make the changes, do you mean change the publish/subscriber flow?

Just in general updating this PR with what we discussed

Reply:
Sure, I'll make those changes.

Reply:
@patrick-5546 @colinli02 Feel free to check over the changes